ssas/hash_test.go

```diff
@@ -65,6 +67,35 @@ func (s *HashTestSuite) TestHashWithIter() {
 	s.True(h1.IsHashOf(val))
 }
 
+func TestHashIterTime(t *testing.T) {
+	// This test is not intended to run with the remain suite.  Is is solely used to determine the number of hash
+	// iterations needed to reach a 1 second duration. To invoke the test use:
+	// DEBUG=true  go test -v github.com/CMSgov/bcda-ssas-app/ssas -run TestHashIterTime
+
+	// Check for local dev environment
```

Comment:
Nit - I don't think the comment here is adding much value. The if statement seems to convey enough

Reply:
Removed